server/handlers/delete_account_oauth.go

```diff
@@ -0,0 +1,59 @@
+package handlers
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/keratin/authn-server/app"
+	"github.com/keratin/authn-server/app/services"
+	"github.com/keratin/authn-server/lib/parse"
+)
+
+func DeleteAccountOauth(app *app.App) http.HandlerFunc {
```

Comment:
it looks like this endpoint is built to delete an arbitrary number of providers at once. this seems more complicated than necessary (the JSON body parsing), and perhaps also not likely to be useful with a typical CRUD-style user interface.

which do you think would be most useful:

1. `DELETE /accounts/:id/oauth` - deletes all oauth records
2. `DELETE /accounts/:id/oauth/:name` - deletes one oauth record

Reply:
Code updated to use 2. I don't have a strong preference, but the second approach indeed aligns more closely with the CRUD-style.